roles/ocp_on_libvirt/tasks/setup.yml

```diff
@@ -47,14 +49,21 @@
       options kvm-{{ proc_family }} ept=1
     dest: /etc/modprobe.d/kvm_nested.conf
     mode: "0644"
-  when: nested_kvm.stdout != 'Y'
+  when: _ool_nested_kvm != 'Y'
```

Comment:
Here you use `_ool_nested_kvm` but you defined it before as `ool_nested_kvm`, `_` is missed at the beginning

Reply:
Fixed